src/binding.cc

```diff
@@ -1476,6 +1483,16 @@ void v8__SnapshotCreator__SetDefaultContext(v8::SnapshotCreator* self,
   self->SetDefaultContext(ptr_to_local(&context), SerializeInternalFields);
 }
 
+size_t v8__SnapshotCreator__AddData(v8::SnapshotCreator* self,
+                                    const v8::Context* context,
+                                    const v8::Data& data) {
+  if (context) {
+    return self->AddData(ptr_to_local(context), ptr_to_local(&data));
+  } else {
+    return self->AddData(ptr_to_local(&data));
```

Comment:
This code path won't work, because it requires v8 to have stores the currently entered isolate in a TLS slot.
But we never enter isolates at all -- so far it's working fine.